### PR TITLE
Adding tests for x-ms-error-response feature

### PR DESCRIPTION
**Pull request by dsgouda**

### CODE REVIEW

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

File: routes/errorStatusCodes.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+};
+
+
+var pathitem = function(coverage) {
+    router.post('/Pets/doSomething/:whatAction', function(req, res, next) {
+        var whatAction = req.params.whatAction;
+        console.log('Inside action: "' + whatAction +'"\n');
+        if (whatAction === 'stay') {
+            coverage['expectedNoErrors']++;
+            res.status(200).send(JSON.stringify(tommyPet));
```
Comment (lmazuel) [inline routes/errorStatusCodes.js]: Please use "end" and not "send", this was the bug that was returning HTML content-type

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Pull request merged**